Packs/MapPattern/ReleaseNotes/1_0_6.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+- **MapPattern**
```

Comment:
```suggestion
##### MapPattern
```
Just this one small fix @spearmin10 
Sorry for the misunderstanding and thx again!

Reply:
@YuvHayun OK, fixed it.